config/sys.config

```diff
@@ -48,8 +48,19 @@
    {colored, true},
    {handlers,
     [
-     {lager_file_backend, [{file, "console.log"}, {level, info}]},
-     {lager_file_backend, [{file, "error.log"}, {level, error}]}
+     {lager_console_backend, [{level, info}]},
+     {lager_file_backend, [
+       {file, "console.log"},
+       {level, info},
+       {size, 10485760},
+       {date, "$D0"},
+       {count, 10}]},
+     {lager_file_backend, [
+       {file, "debug.log"},
+       {level, debug},
+       {size, 10485760},
+       {date, "$D0"},
+       {count, 10}]}
```

Comment:
NB: this allows headless / dockerized app logs to be introspected if needed. Maybe unnecessary.